openapi.json

```diff
@@ -280,6 +280,110 @@
           }
         }
       }
+    },
+    "/v2/{ocpVersion}/gathering_rules": {
+      "summary": "Return remote configuration base on OCP version",
+      "description": "This endpoint returns the specific configuration used by Insights Operator based on version of the cluster where the operator runs.",
+      "get": {
+        "parameters": [
+          {
+            "name": "ocpVersion",
+            "description": "Version of the OCP cluster",
+            "schema": {
+              "type": "string"
+            },
+            "in": "path",
+            "required": true,
+            "example": "1.0.0"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "version": {
+                      "type": "string"
+                    },
+                    "conditional_gathering_rules": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "conditions": {
+                            "type": "array",
+                            "items": {
+                              "type": "object",
+                              "properties": {
+                                "alert": {
+                                  "type": "object",
+                                  "properties": {
+                                    "name": {
+                                      "type": "string"
+                                    }
+                                  },
+                                  "required": [
+                                    "name"
+                                  ]
+                                },
+                                "type": {
+                                  "type": "string"
+                                }
+                              },
+                              "required": [
+                                "alert",
+                                "type"
+                              ]
+                            }
+                          },
+                          "gathering_functions": {
+                            "type": "object"
```

Comment:
I took this from v1 endpoint, but can try to dig up more about this field.

Reply:
I quickly searched through [conditions JSONs](https://github.com/RedHatInsights/insights-operator-gathering-conditions/tree/main/conditions) and it seems that content in this field can vary a lot. I am not sure if we should specify more, it would probably get quite extensive.